src/helpers/compose.ts

```diff
@@ -0,0 +1,22 @@
+type Func<I, O> = (input: I) => O;
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+type Compose = (...functions: Func<any, any>[]) => Func<any, any>;
+
+/**
+ * Performs right-to-left function composition. The arguments
+ * should be unary function
+ *
+ * **Note:** The result of compose is not automatically curried.
+ *
+ * @sig ((y -> z), (x -> y), ..., (o -> p), ((a, b, ..., n) -> o)) -> ((a, b, ..., n) -> z)
+ * @param {...Function} ...functions The functions to compose
+ * @symb compose(f, g, h)(x) = f(g(h(x)))
+ */
+
+const compose: Compose =
```

Comment:
You've implemented this function but you didn't use it. why?

Reply:
it will be useful, i probably use it in the next modules that i implement or ...

Reply:
Add it whenever you need it.